bin/add_celltypes_to_sce.R

```diff
@@ -250,6 +267,47 @@ if (!is.null(opt$cellassign_predictions)) {
     }
     metadata(sce)$cellassign_reference_organs <- cellassign_organs
   }
+} else {
+  has_cellassign <- FALSE # use to decide if we should assign consensus labels
+}
```

Comment:
With `has_cellassign` already set, we shouldn't need this.
```suggestion
} 
```